### ISSUE TITLE
[1.181127.0] Snapshot tab, give an option to check for TB event, or Rare Units

### ISSUE THREAD

**Issue by DarkPhoenixSniper** [PopGoesTheWza/swgoh-tb-sheets#32]
I am suggesting to have an option to check for the TB characters, or Event characters.
If Event is selected.  Check for JTR, JTR Required, BH's, Chewbacca, Nest, etc...  Maybe customizable list on the Meta sheet.  Listed in the order the user wants it shown.

**Comment by PopGoesTheWza** [comment]
By event characters, do you the legendary events? If so what you suggest is some readyness assessment?

**Comment by DarkPhoenixSniper** [comment]
@PopGoesTheWza correct... Essentially a snapshot of what you are looking for that could be defined by the user.  I.e. I want to see if this guy has JTR, or if not, has all requirements.
JTR
Finn
Rey Scav
BB-8
Vet Han
Vet Chewie
listed out the same way for the TB characters.

But then if I want to add new ones in, like BH's for Chewie, or OR I can simply add them to the list on META in the order I want them shown.

**Comment by PopGoesTheWza** [comment]
@DarkPhoenixSniper sort of like the Snapshot tab with an editable list of Units to detail?

**Comment by PopGoesTheWza** [comment]
Silly. I had overlooked the clear issue title :)

**Comment by DarkPhoenixSniper** [comment]
while still using the Snapshot tab.  but just simply a dropdown that would allow me to say Event/Rare (or whatever you decide to call it) with a customizable list on the META sheet. (Logically located here along with the TB's for Light/Dark.  Might make it easier to see what the guy has/is ready for instead of searching through his entire roster.
